test/test_train_mnist.py

```diff
@@ -78,7 +78,7 @@ def train_mnist():
     devices = [':{}'.format(n) for n in range(0, FLAGS.num_cores)]
     inputs = torch.zeros(FLAGS.batch_size, 1, 28, 28)
     target = torch.zeros(FLAGS.batch_size, dtype=torch.int64)
-    xla_model = xm.XlaModel(model, [inputs], loss_fn=F.nll_loss, target=target,
+    xla_model = xm.XlaModel(model, (inputs), loss_fn=F.nll_loss, target=target,
```

Comment:
This does not look like correct.
(input) == input
Does not create a tuple.
We should be OK with a list ([]) here.
Or tuple([inputs])

Reply:
the right way is `(inputs, )`

Reply:
We shouldn't need a tuple at all here, just `inputs` would suffice.

Reply:
Actually no, `TestReplicatedSum` is the only one which would break. I'll try another way.